lib/docx/body-reader.js

```diff
@@ -133,14 +134,27 @@ function BodyReader(options) {
             return readXmlElements(element.children)
                 .map(function(children) {
                     var properties = _.find(children, isRunProperties);
-                    
-                    return new documents.Run(
-                        children.filter(negate(isRunProperties)),
-                        properties
-                    );
+                    var filteredChildren = children.filter(negate(isRunProperties));
+
+                    if (properties && properties.styleName === 'Hyperlink' && lastInstrText) {
+                        return new documents.Hyperlink(filteredChildren, {href: parseHyperlink(lastInstrText)});
```

Comment:
I think this breaks the behaviour of runs since it replaces the run with a hyperlink, rather than creating a run that contains a hyperlink. For instance, this would mean that bold formatting would be ignored.